__tests__/dataframe.test.ts

```diff
@@ -1314,8 +1314,8 @@ describe("dataframe", () => {
     ]);
     expect(actual).toFrameEqual(expected);
   });
-  test("pivot", () => {
```

Comment:
Any reason for removing this? Thx

Reply:
gah, accidently removed when adding scopes, will amend and push again